src/tribler-common/tribler_common/tests/test_patch_import.py

```diff
@@ -15,11 +15,18 @@ async def test_mock_import_mocked_lib():
     import library_that_does_not_exist
     assert library_that_does_not_exist
 
+    # test that mocks for inner inner function are presented
+    assert library_that_does_not_exist.inner_function
+
+    # test that magic methods of objects from patched library works correctly
+    assert len(library_that_does_not_exist.inner_set) == 0
+
 
 @patch_import([])
 async def test_mock_import_import_real_lib():
     with pytest.raises(ImportError):
         import library_that_does_not_exist
+        library_that_does_not_exist.inner_function()
```

Comment:
This call is unnecessary for the test itself, but this line prevents removing "unused" `import library_that_does_not_exist` during IDE autoformatting.